src/QMCDrivers/QMCCostFunction.cpp

```diff
@@ -290,7 +290,19 @@ void QMCCostFunction::checkConfigurations()
         //allocate vector
         std::vector<Return_t> Dsaved(NumOptimizables, 0.0);
         std::vector<Return_t> HDsaved(NumOptimizables, 0.0);
+
+        std::vector<Return_ct> cDsaved(NumOptimizables, 0.0);
+        std::vector<Return_ct> cHDsaved(NumOptimizables, 0.0);
+
+        #ifndef QMC_COMPLEX
```

Comment:
Need a FIXME note for every place which will be fixed later.

Reply:
I have added a FIXME note.